src/functions.php

```diff
@@ -597,11 +597,18 @@ function warning(string $message): void
     }
 }
 
+/**
+ * Writes a message to the output.
+ */
+function write(string|iterable $message, int $options = 0): void
```

Comment:
It was removed as it only redirects call to output() and don’t add any helpers beside parse.

Reply:
`write()` is more convenient and readable than `output()->write()`. It does not make sense to me to break backwards compatibility here.

Reply:
writeln also adds host info now, but write not.

Reply:
Yes, but it can't, because the _whole point_ is that it doesn't _necessarily_ start a new line. I agree we might want a different solution here. Maybe we need two functions, like `writeStart()` and `writeContinue()`, where only _start_ adds the host info and _continue_ does not, but neither adds a trailing newline. Open to ideas.